docs/swagger.yaml

```diff
@@ -654,117 +841,95 @@ definitions:
-          \ has been approved/rejected in the Vipps mobile application."
-        example: "https://api.merchant.no/landing"
-      price:
+      amount:
         type: "integer"
         format: "int32"
         description: "Treated as a whole sum postfixed with two decimals for cents,\
           \ eg 234 = 2.34 NOK"
-        example: 7900
-      productDescription:
+        example: 49900
+      amountRefunded:
+        type: "integer"
+        format: "int32"
+        description: "The total amount which has been refunded, in case of status\
+          \ refund/partial refund.\nTreated as a whole sum postfixed with two decimals\
+          \ for cents, eg 234 = 2.34 NOK"
+        example: 49900
+      description:
         type: "string"
-        example: "Fin abonnementspakke"
-      productName:
+        description: "Description of the charge"
+        example: "Premier League subscription: September"
+      due:
         type: "string"
-        description: "Name of the product being subscribed to."
-        example: "Pluss-abonnement"
-  DraftAgreementResponse:
-    type: "object"
-    properties:
-      agreementResource:
+        format: "date"
+        description: "The due date for this charge"
+        example: "2030-12-31"
+      id:
         type: "string"
-        description: "Reference to AgreementDto which user may agree to. Initially\
-          \ the AgreementDto is in a pendingUserApproval state, and it enters active\
-          \ state once user has approved in the Vipps application."
-        example: "https://api.vipps.no/api/v1/agreement/agr_5kSeqzFAMkfBbc"
-      agreementId:
+        description: "Identifier for this charge (for this customer's subscription)."
+        default: "NOT_SET"
+        maxLength: 15
+        example: "chg_WCVbcAbRCmu2zk"
+      status:
         type: "string"
-        description: "Id of a AgreementDto which user may agree to. Initially the\
-          \ AgreementDto is in a pendingUserApproval state, and it enters active state\
-          \ once user has approved in the Vipps application."
-        example: "agr_5kSeqzFAMkfBbc"
-      redirectToVipps:
+        default: "PENDING"
+        enum:
+        - "PENDING"
+        - "DUE"
+        - "CHARGED"
+        - "FAILED"
+        - "CANCELLED"
+        - "PARTIALLY_REFUNDED"
+        - "REFUNDED"
+        - "PROCESSING"
+        example: "PENDING"
+      transactionId:
         type: "string"
-        description: "Customer should be redirected to Vipps using this URL."
-        example: "https://api.vipps.no/api/v1/register/U6JUjQXq8HQmmV"
-  FieldErrorDto:
+        description: "Contains null until the status has reached CHARGED"
+        maxLength: 36
+        example: "5001419121"
+      type:
+        type: "string"
+        default: "RECURRING"
+        enum:
+        - "INITIAL"
+        - "RECURRING"
+        example: "RECURRING"
+  Error:
     type: "object"
+    required:
+    - "message"
     properties:
```

Comment:
Required field. Maybe make a correlationId (contextId)?

Reply:
`contextId`, please! After a quick check, I found out that `correlationId` is a bit stigmatized in Vipps. Ping @niklud